### ISSUE TITLE
Document the right backup/restore procedure

### ISSUE THREAD

**Issue by travlr** [regisb/tutor#186]
Primary need is to restore courses' content from a backup-copy made of the TUTOR_ROOT

## Steps to reproduce

1) Make a backup of TUTOR_ROOT directory.
2) Create a new tutor instance (via `tutor local quickstart`).
3) Copy `data` and `config.yml` from backup directory to new TUTOR_ROOT (using `sudo`).
4) `tutor local start` produces errors 

[log.txt](https://github.com/regisb/tutor/files/2992051/log.txt)

## Additional info (IMPORTANT)

Tutor Version: tutor, version 3.2.1

**Comment by regisb** [comment]
Is there an `env` folder inside the TUTOR_ROOT? if not, you need to generate it with `tutor config save --silent`.

For a full re-build, you should probably run:

    tutor images pull all
    tutor local quickstart

**Comment by travlr** [comment]
Hi Regis, and thank you. I do have an `env` folder present in the current `TUTOR_ROOT`. I am still somewhat confused by the procedure, in general. Does it matter if the `tutor` instance is a new instance or not. Could you also include the "general" procedure in the docs when you are able. ~Peter

**Comment by travlr** [comment]
I was able to restore the courses Regis, thank you for taking the time for me! I had to create a new instance via `quickstart` again, then I made sure the new instance had an `env` dir. I copied the `data` dir and the `config.yml` from the backup to the new instance's `TUTOR_ROOT`.

**Comment by regisb** [comment]
Yes, I'll update the documentation soon. In a nutshell, this is what the procedure looks like:

## Backup

    sudo cp -r --preserve $(tutor config printroot)/config.yml $(tutor config printroot)/data ~/backup/

## Restore

   sudo cp -r --preserve ~/backup/config.yml ~/backup/data $(tutor config printroot)/
   tutor local quickstart --pullimages

**Comment by crivet** [comment]
Just for clarification, does this specified process allow for copying the data from one tutor instance to another?

i.e. Is it possible to generate the backup, move the directory to another machine running tutor, restore the backup there? or is there another process for transferring data between tutor builds?

I ask because this would be helpful in setting up test cases between development instances.

**Comment by regisb** [comment]
> i.e. Is it possible to generate the backup, move the directory to another machine running tutor, restore the backup there? or is there another process for transferring data between tutor builds?

Yes, absolutely. I confess I did not try it myself, but I see no reason that it shouldn't work.

**Comment by travlr** [comment]
Hi @crivet .. The proceedure I use to move between machines involves creating a compressed archive, as I've described in issue https://github.com/regisb/tutor/issues/188 . 

I have found that moving the data to another machine worked for me, but I do get an error as described in the issue linked above. This error doesn't seem to affect my installation as my needs aren't currently using search or the forums. 

I do want to address this issue as soon as I have time.
